.github/workflows/cppcheck-monitor.yaml

```diff
@@ -1,13 +1,12 @@
-name: cppcheck-all
+name: cppcheck-monitor
```

Comment:
```suggestion
name: cppcheck-daily
```

This way the scope, schedule and function will be more clear, what do you think?

Reply:
Agreed, thanks for the proposal.